File: FaceMe_Android/src/com/example/faceme_android/PosterEntity.java
package com.example.faceme_android;

import android.graphics.Bitmap;



public class PosterEntity {

	private long key;
	private String originalPosterKey;
	private String thumbnailKey;
	private String nonfacePosterKey;
	private String movieName;
	private String classification;
	private String posterName;
	
	
	public Bitmap thumbnail;
	public Bitmap originalPoster;
	public Bitmap nonfacePoster;

	public PosterEntity(long key, String originalPosterKey,
			String thumbnailKey, String nonfacePosterKey, String movieName,
			String classification, String posterName) {
		super();
		this.key = key;
		this.originalPosterKey = originalPosterKey;
		this.thumbnailKey = thumbnailKey;
		this.nonfacePosterKey = nonfacePosterKey;
		this.movieName = movieName;
		this.classification = classification;
		this.posterName = posterName;
	}

	public long getKey() {
		return key;
	}


	public String getOriginalPosterKey() {
		return originalPosterKey;
	}


	public String getThumbnailKey() {
		return thumbnailKey;
	}


	public String getNonfacePosterKey() {
		return nonfacePosterKey;
	}


	public String getMovieName() {
		return movieName;
	}


	public String getClassification() {
		return classification;
	}

	public String getPosterName() {
		return posterName;
	}
}

File: FaceMe_iOS/FaceMe_iOS/PosterEntity.m
//
//  PosterEntity.m
//  FaceMe_iOS
//
//  Created by ShawnWu on 11/26/13.
//  Copyright (c) 2013 Bo Pang. All rights reserved.
//

#import "PosterEntity.h"

@implementation PosterEntity
@synthesize key,orginalPoster,originalPosterKey,thumbnail,thumbnailKey,movieName,nonfacePoster,nonfacePosterKey,posterName,classification,faces;
-(id)initWithKey:(NSString*) key1 AndOP:(NSString*)orginalPosterKey1 AndTK:(NSString*)thumbnailKey1 AndNFK:(NSString*)nonfaceKey1 AndM:(NSString*)movieName1 AndC:(NSString*)classification1 AndPN:(NSString*)posterName1{
    if (self=[super init]) {
        self.key=key1;
        self.originalPosterKey=orginalPosterKey1;
        self.thumbnailKey=thumbnailKey1;
        self.nonfacePosterKey=nonfaceKey1;
        self.movieName=movieName1;
        self.classification=classification1;
        self.posterName=posterName1;
        
    }
    return self;
}
//-(id)init:{
//    return self;
//}
@end
